apps/web/src/components/TalentNominationEventCard/TalentNominationEventCard.tsx

```diff
@@ -1,31 +1,45 @@
 import { useIntl } from "react-intl";
 
 import { Heading, CardBasic, Link } from "@gc-digital-talent/ui";
-import { formatDate } from "@gc-digital-talent/date-helpers";
+import { formatDate, parseDateTimeUtc } from "@gc-digital-talent/date-helpers";
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
 
-interface TalentNominationEventCardProps {
-  mode: "Current"; // more modes coming soon.
-  communityName: string;
-  title: string;
-  openDate: Date;
-  closeDate: Date;
-  description: string;
-  startUrl: string;
-  learnMoreUrl?: string;
+export interface TalentNominationEventCardProps {
```

Comment:
Looks like we can drop the export here.

Reply:
3b1bc392e15b90730efa050fc48b7e0ac0c7ef9c